Examples/Run/Reconstruction/Common/SeedingExample.cpp

```diff
@@ -121,21 +121,21 @@ int runSeedingExample(int argc, char* argv[],
   };
   seedingCfg.outputSeeds = "seeds";
   seedingCfg.outputProtoTracks = "prototracks";
-  seedingCfg.rMax = 100.;
-  seedingCfg.deltaRMax = 60.;
-  seedingCfg.collisionRegionMin = -250;
-  seedingCfg.collisionRegionMax = 250.;
-  seedingCfg.zMin = -2000.;
-  seedingCfg.zMax = 2000.;
+  seedingCfg.rMax = 100._mm;
+  seedingCfg.deltaRMax = 60._mm;
+  seedingCfg.collisionRegionMin = -250_mm;
+  seedingCfg.collisionRegionMax = 250._mm;
+  seedingCfg.zMin = -2000._mm;
+  seedingCfg.zMax = 2000._mm;
   seedingCfg.maxSeedsPerSpM = 1;
   seedingCfg.cotThetaMax = 7.40627;  // 2.7 eta
   seedingCfg.sigmaScattering = 50;
```

Comment:
```suggestion
  seedingCfg.sigmaScattering = 5;
```

I know it's not really related...

Reply:
I think I'd prefer doing this in it's own PR to decouple it from the purely technical changes here.